### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by bitbonk**
Just a small typo fix. I hope you don’t mind.

### CODE REVIEW

**Review by bigludo7 — DISMISSED** [review]
Thanks

**Comment by PedroDiez** [comment]
Thanks, and will take advantage of some point pending to be aligned (and follow up of fulfilment of goods). It was clarified under this issue: https://github.com/camaraproject/CarrierBillingCheckOut/issues/45

https://github.com/camaraproject/CarrierBillingCheckOut/issues/45#issuecomment-1403710655

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Repository to describe, develop, document and test the CarrierBillingCheckOut AP
 
 ## Notice for the reader
 The Carrier Billing family features 2 APIs with **distinct business scope**:
-* 'Carrier billing checkout' : Service that provides functionality for enabling the first the purchase and then payment request, and follow up of fulfilment of goods against OB Carrier Billing Systems.
+* 'Carrier billing checkout' : Service that provides functionality for enabling first the purchase and then the payment request, and follow up of fulfilment of goods against OB Carrier Billing Systems.
```
Comment (PedroDiez) [inline README.md:13]:
```suggestion
* 'Carrier billing checkout' : Service that provides functionality for enabling first the purchase and then the payment request against OB Carrier Billing Systems.
```

**Review by PedroDiez — APPROVED** [review]
LGTM

**Review by bigludo7 — APPROVED** [review]
Good for me. Thanks !

**Comment by PedroDiez** [comment]
Merging it, thanks @bitbonk , @bigludo7 !!

**Pull request merged**